### ISSUE TITLE
Apache Upload Exploit

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bdshadow/gwtupload#190]
```
What steps will reproduce the problem?
1.See exploit details 
http://blog.spiderlabs.com/2014/02/cve-2014-0050-exploit-with-boundaries-loops-w
ithout-boundaries.html

What is the expected output? What do you see instead?
Exploit pegs CPU @100% in infinite loop.

What version of the product are you using? On what operating system?
Affects Tomcat 7 and Tomcat 8. Must update upload library to 1.3.1.

Please provide any additional information below.

```

Original issue reported on code.google.com by `lifeofl...@gmail.com` on 24 Feb 2014 at 10:32

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Library is currently using Common File Upload 1.2.1
```

Original comment by `lifeofl...@gmail.com` on 24 Feb 2014 at 10:34

**Comment by GoogleCodeExporter** [comment]
Original comment by `manuel.carrasco.m` on 6 Mar 2014 at 5:02
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 5741a197ca47.
```

Original comment by `manuel.carrasco.m` on 2 Apr 2014 at 8:18
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `manuel.carrasco.m` on 28 Apr 2014 at 6:27
- Added labels: **Milestone-1.0.1**
